sw/device/silicon_creator/lib/drivers/keymgr.c

```diff
@@ -56,6 +56,9 @@ static rom_error_t expected_state_check(uint32_t expected_state) {
 
   // Read the working state with sec_mmio so that we can check the expected
   // value periodically.
+  // Note: Any checks in ROM_EXT after a keymgr transition must be performed
```

Comment:
WDYT about promoting these notes to function comments?

Reply:
Good point, I'll also mention that `keymgr_check_state()` already handles this.